<launch>

  <group ns="test_controller" clear_params="true">
    <param name="type" type="string" value="SineSweepController" />
    <param name="joint_name" type="string" value="head_tilt_joint" />

    <!-- Analysis params -->
    <param name="first_mode" type="double" value="14" />
    <param name="second_mode" type="double" value="19" />
    <param name="error_tolerance" type="double" value="0.15" />
    
    <!-- Sine sweep params -->
    <param name="amplitude" type="double" value="5.0" />
    <param name="start_freq" type="double" value="5.0" />
    <param name="end_freq" type="double" value="100.0" />
    <param name="duration" type="double" value="10.0" />
  </group>
    
  
  <node pkg="mechanism_control" type="spawner.py" args="test_controller" />
  
  <node pkg="qualification" type="hysteresis_sinesweep_plot.py" />
</launch>
